packages/app/src/pages/Specs/Runner.vue

```diff
@@ -31,6 +31,7 @@ import SpecRunnerContainerOpenMode from '../../runner/SpecRunnerContainerOpenMod
 import SpecRunnerContainerRunMode from '../../runner/SpecRunnerContainerRunMode.vue'
 import { useEventManager } from '../../runner/useEventManager'
 import { useSpecStore } from '../../store'
+import { isRunMode } from '@packages/frontend-shared/src/utils/isRunMode'
```

Comment:
I don't think this belongs in `frontend-shared` - the concept of open/run mode is exclusively to the App. Any particular reason to put in in `frontend-shared`?